instance.go

```diff
@@ -286,12 +285,32 @@ func (i *Instance) DetermineSecondaryJournalDirectory() (string, error) {
 	return "", fmt.Errorf("unable to determine secondary journal directory")
 }
 
+// DetermineISCDatFileName returns the filename of the InterSystems DAT files used by the instance
+func (i *Instance) DetermineISCDatFileName() string {
+	switch i.Product {
+	case Iris:
+		return "IRIS.DAT"
```

Comment:
I'm not going to repeat this comment everywhere but I would likely have made any strings like this constants. It is certainly debatable though.